turnserver/config/config.go

```diff
@@ -0,0 +1,99 @@
+package config
```

Comment:
is it necessary to have all these sub-package especially as most have but a single file.